core_lib/canvasrenderer.cpp

```diff
@@ -80,7 +80,6 @@ void CanvasRenderer::paint( Object* object, int layer, int frame, QRect rect )
     QPainter painter( mCanvas );
 
     painter.setWorldTransform( mViewTransform );
-    painter.setRenderHint( QPainter::SmoothPixmapTransform, mOptions.bAntiAlias );
```

Comment:
Curious why remove this line? It's an option that you can switch it on and off in Preferences

Reply:
Set renderHint with smoothPixmapTransform serves no purpose here, it creates a similar effect to that of mipmaps for textures, making the output look blurry which is not an effect you would want for pixel drawing application. The anti-aliasing checkbox in preferences should only affect the edges of vector lines (which it does)

![mipmap](https://user-images.githubusercontent.com/1045397/30632429-50f22ed8-9de8-11e7-87df-863485d4d55d.gif)

notice the output becoming blurry as soon as i let go of the mouse button.

Reply:
Alright, you convinced me. 
Going to merge it.